torch_xla/dynamo_bridge.py

```diff
@@ -0,0 +1,317 @@
+import dataclasses
```

Comment:
let's move this file into `torch_xla/core` maybe? we generally don't have python file under `torch_xla` dir directly.